lib/request-proxy.js

```diff
@@ -26,6 +26,8 @@ var TypedError = require('error/typed');
 
 var safeParse = require('./util').safeParse;
 
+var RETRY_SCHEDULE = [0, 0.5, 2];
```

Comment:
Is doing a retry with 0 delay bad? It's probably not good to do an immediate retry on failure. Not sure what a good delay is though.